src/redux/books/index.js

```diff
@@ -0,0 +1,45 @@
+// books.js
+
+// Actions
+const CREATE = 'CREATE';
+const UPDATE = 'UPDATE';
+const DELETE = 'DELETE';
+const LOAD = 'LOAD';
```

Comment:
- [ ] **Required:** You are required to use `duck's pattern` in defining the actions of the reducer. [Read more.](https://github.com/erikras/ducks-modular-redux)